test/distributed/org/apache/cassandra/distributed/impl/IsolatedExecutor.java

```diff
@@ -282,7 +282,7 @@ public static <T> T waitOn(Future<T> f)
         }
         catch (ExecutionException e)
         {
-            throw Throwables.throwAsUncheckedException(e.getCause());
+            throw new RuntimeException(e.getCause());
```

Comment:
asked in the dev list, no one complained... this will break tests so shouldn't be done in this patch... will revert when testing is done and file a ticket against our other branches